src/vertex_tools.rs

```diff
@@ -32,25 +31,44 @@ pub fn to_barycentric(
     }
 }
 
+/// Checks if all three points of a triangle lay on the same line.
+pub fn are_triangle_vertices_colinear(
```

Comment:
Nit: `are_vertices_colinear`? Also update comment if you change the name

Reply:
just an explanation: the input argument is a tuple `(pt, pt, pt)`, which is used in triangle-related functions